cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java

```diff
@@ -570,6 +573,50 @@ private void _convertToWarProject(File warsDir, File pluginDir) {
 				ivyFile.delete();
 			}
 
+			File liferayPluginPackageFile =
```

Comment:
Can you add unit tests for this behavior?

Reply:
will do soon